src/Components/BookList/BookDisplay.js

```diff
@@ -2,40 +2,65 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../../redux/books/books';
-import './BookList.css';
+import './BookDisplay.css';
+import './circle.css';
 
 const BookDisplay = ({ bookList }) => {
   const dispatch = useDispatch();
+  const percentage = Math.floor(Math.random() * 100);
+  const textToPass = `${percentage}%`;
 
   return (
     <div>
       {bookList.map((book) => (
         <div key={book.item_id}>
-          <div className="container bookContainer py-4">
+          <div className="container bookContainer py-4 my-3">
             <div className="row totalRow">
               <div className="col-5">
                 <h5 className="col-12 booktype">{book.category}</h5>
                 <h2 className="col-12 bookTitle">{book.title}</h2>
-                <h5 className="col-12 author">{book.author}</h5>
+                <h5 className="col-12 bookAuthor">{book.author}</h5>
+                <div className="col-12">
+                  <button className="col-3 my-2 removeButton comment" type="button">
+                    Comments
+                    <span>|</span>
+                  </button>
+                  <button
+                    className="col-3 my-2 mx-2 removeButton"
+                    type="button"
+                    id={book.item_id}
+                    onClick={(event) => dispatch(removeBook(event.target.id))}
+                  >
+                    Remove
+                    <span>|</span>
+                  </button>
+                  <button type="button" className="col-3 my-2 removeButton">
+                    Edit
+                  </button>
+                </div>
               </div>
```

Comment:
[REQUIRED] Kindly follow the instructions in the image below

![image](https://user-images.githubusercontent.com/62938735/194713044-3e9de44d-02c1-461e-ae34-925c5a76e0e6.png)